### PR TITLE
Created getCluster function and updated 1 test to reflect changes

### PR DESCRIPTION
**Pull request by akarukappadath**
This is the first of a series of PRs for the getCluster function and resulting test updates. This PR has been opened to share initial ideas of how this function will be structured, and for feedback on how the test was updated. 

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/test/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponentSpec.scala [hunk 1]
```diff
@@ -23,81 +23,27 @@ class ClusterComponentSpec extends TestComponent with FlatSpecLike with CommonTe
...
+                                       status = ClusterStatus.Creating)
+
+    val c3 = getCluster(3).copy(auditInfo = auditInfo.copy(dateAccessed = dateAccessed),
+                                       machineConfig = MachineConfig(Some(3),Some("test-master-machine-type"), Some(500), Some("test-worker-machine-type"), Some(200), Some(2), Some(1)),
+                                       serviceAccountInfo = ServiceAccountInfo(None, Some(serviceAccountEmail)),
+                                       status = ClusterStatus.Running)
+
+    val c4 = getCluster(4).copy(clusterName = c1.clusterName,
+                                       googleProject = c1.googleProject,
+                                       dataprocInfo = dataprocInfo.copy(operationName = Option(OperationName("op3")), stagingBucket = Some(GcsBucketName("testStagingBucket4"))))
```
Comment (kyuksel) [inline src/test/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponentSpec.scala]: Should `c4` use `OperationName("op4")` for consistency?

File: src/test/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponentSpec.scala [hunk 2]
```diff
@@ -340,7 +269,8 @@ class ClusterComponentSpec extends TestComponent with FlatSpecLike with CommonTe
   }
 
   it should "get list of clusters to auto freeze" in isolatedDbTest {
```
Comment (kyuksel) [inline src/test/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponentSpec.scala]: I'm guessing the lines starting with `Cluster(` below got misaligned by IntelliJ?

Comment (akarukappadath) [reply src/test/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponentSpec.scala]: Yup, I think so. I was thinking this won't really make a difference in the end once they're all replaced (hopefully soon)

**Comment by akarukappadath** [comment]
Some git struggles - moving to new PR https://github.com/DataBiosphere/leonardo/pull/539

**Pull request closed without merging**